src/Sylius/Bundle/OrderBundle/Aggregator/AdjustmentsAggregator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\OrderBundle\Aggregator;
+
+use Sylius\Component\Order\Model\AdjustmentInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class AdjustmentsAggregator implements AdjustmentsAggregatorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function aggregate(array $adjustments)
+    {
+        $aggregatedAdjustments = array();
+        foreach ($adjustments as $adjustment) {
+            if (!$adjustment instanceof AdjustmentInterface) {
```

Comment:
We may extract this assertion to some private method or even assert the whole array before this loop.